tests/sentry/monitors/logic/test_mark_failed.py

```diff
@@ -369,12 +396,16 @@ def test_mark_failed_with_missed_reason_issue_platform(self, mock_produce_occurr
         last_checkin = timezone.now()
         expected_time = monitor.get_next_expected_checkin(last_checkin)
 
-        assert mark_failed(
-            monitor_environment,
-            last_checkin=last_checkin,
-            reason=MonitorFailure.MISSED_CHECKIN,
-            occurrence_context={"expected_time": expected_time.strftime(SUBTITLE_DATETIME_FORMAT)},
+        failed_checkin = MonitorCheckIn.objects.create(
+            monitor=monitor,
+            monitor_environment=monitor_environment,
+            project_id=self.project.id,
+            status=CheckInStatus.MISSED,
+            date_added=last_checkin,
+            expected_time=expected_time,
+            duration=monitor.config.get("max_runtime"),
         )
+        assert mark_failed(failed_checkin, ts=last_checkin)
```

Comment:
don't we pass `ts=None` for missed check-ins currently?

Reply:
Check line 374, it was called `last_checkin`

Reply:
i guess I meant in the actual task. tests might be wrong here

Reply:
it's fine, i'll fix them all up in #55395